src/main/java/vendored/org/apache/dolphinscheduler/common/utils/process/ProcessImplForWin32.java

```diff
@@ -95,7 +95,8 @@
 
     private static final int OFFSET_WRITE = 1;
 
-    private static final WinNT.HANDLE JAVA_INVALID_HANDLE_VALUE = new WinNT.HANDLE(Pointer.createConstant(-1));
+    private static final WinNT.HANDLE JAVA_INVALID_HANDLE_VALUE =
+            new WinNT.HANDLE(Pointer.createConstant(POINTER_SIZE == 8 ? -1L : 4294967295L));
```

Comment:
`4294967295L` is too magical. Can that be hoisted up as a static and then link to some docs?